cascade/src/main/java/me/saket/cascade/CascadePopupMenu.kt

```diff
@@ -69,6 +75,31 @@ open class CascadePopupMenu @JvmOverloads constructor(
       popup.contentView.background = it
     }
 
+    styler.overlayColor()?.let {
+      check(context is Activity) { "Activity context is required in order to add an overlay view" }
+      val container = (context.window.decorView as ViewGroup)
+      val overlay = View(context, null, 0).apply {
+        alpha = 0f
+        setBackgroundColor(it)
+      }
+      container.addView(overlay)
+
+      val alphaInAnimator = ObjectAnimator.ofFloat(overlay, View.ALPHA, 1f).apply {
+        duration = 500
+        interpolator = DecelerateInterpolator()
+      }
+      alphaInAnimator.start()
+
+      popup.setOnDismissListener {
+        val alphaOutAnimator = ObjectAnimator.ofFloat(overlay, View.ALPHA, 0f)
```

Comment:
Any reason for not using `overlay.animate().alpha(0f)` instead of an `ObjectAnimator`?